native/dll/DShowPlayer-2/GraphPluginHandler.cpp

```diff
@@ -30,7 +30,7 @@ GraphPluginHandler::GraphPluginHandler(void)
 	
 	dwLen = GetModuleFileName(NULL, szModulePath, MAX_PATH);
 	pch=strrchr(szModulePath,'\\')+1;
-	dwLen = dwLen - strlen(pch);
+	dwLen = dwLen - (int)strlen(pch);

```

Comment:
For code that is Windows specific, I *think* you can just use the lstrlenA function, and that returns an int and then you can avoid casts...that then aligns with all the other win calls which are expecting int for sizes rather than size_t.

Reply:
I'll try that

Reply:
Done. There's some other casts that I could tidy up with this too